packages/esm-patient-search-app/src/patient-search-page/patient-banner/ui-components/overflow-menu.component.tsx

```diff
@@ -43,49 +45,25 @@ const CustomOverflowMenuComponent: React.FC<CustomOverflowMenuComponentProps> =
   }, [wrapperRef]);
 
   return (
-    <div
-      data-overflow-menu
-      className="cds--overflow-menu"
-      style={{
-        width: 'auto',
-        height: 'auto',
-      }}
-      ref={wrapperRef}>
-      <Button
```

Comment:
I don't understand why this changes from a Carbon `button` to a standard `button` element?